doc/src/guides/solving/solve-diophantine-equation.md

```diff
@@ -0,0 +1,279 @@
+```
+
+### Specify the Order of Symbols in the Result
+
+We recommend you specify the order of symbols in the result to avoid confusion.
+Use the `syms` parameter and pass it a tuple or list of symbols to ensure the
+result will be in that order, for example `syms=my_syms`, as in the examples on
+this page.
+
+### Limitations
+
+Currently, following five types of Diophantine equations can be solved using
+{meth}`~sympy.solvers.diophantine.diophantine.diophantine` and other helper
+functions of the Diophantine module.
+
+- Linear Diophantine equations: $a_1x_1 + a_2x_2 + \ldots + a_nx_n = b$
+- General binary quadratic equation: $ax^2 + bxy + cy^2 + dx + ey + f = 0$
+- Homogeneous ternary quadratic equation: $ax^2 + by^2 + cz^2 + dxy + eyz + fzx
+  = 0$
+- Extended Pythagorean equation: $a_{1}x_{1}^2 + a_{2}x_{2}^2 + \ldots +
+  a_{n}x_{n}^2 = a_{n+1}x_{n+1}^2$
+- General sum of squares: $x_{1}^2 + x_{2}^2 + \ldots + x_{n}^2 = k$
+
+## Use the Solution Result
+
+### Extract Expressions From the Result
+
+{func}`~.diophantine` returns results as a set of tuples, where each element in
+a tuple is an expression for a variable in your equation. For example, for the
+Pythogorean equation, the result is a set containing one tuple where the
+expressions correspond to (a, b, c). That is, the tuple represents `a = 2*p*q, b
+= p**2 - q**2, c = p**2-q**2`. Because you cannot extract an element (here, a
+tuple) from a set by subscripting the set, you can create a dictionary of
+symbol-expression pairs to extract an expression by its symbol:
+
+```py
+>>> from sympy.solvers.diophantine import diophantine
+>>> from sympy import symbols
+>>> a, b, c = symbols("a, b, c", integer=True)
+>>> my_syms = (a, b, c)
+>>> pythag = a**2 + b**2 - c**2
+>>> solution, = diophantine(pythag, syms=my_syms)
+>>> solution
+(2*p*q, p**2 - q**2, p**2 + q**2)
+>>> # Convert set to list
+>>> solution_dict = dict(zip(my_syms, solution))
+>>> solution_dict
+{a: 2*p*q, b: p**2 - q**2, c: p**2 + q**2}
+>>> # Extract an expression for one variable using its symbol, here a
+>>> solution_dict[a]
+2*p*q
+```
+
+Less elegantly, you can convert the set to a list, and then subscript the list.
+It is a common mistake to forget the order of parameters, so this method is more
+prone to errors:
+
+```py
+>>> from sympy.solvers.diophantine import diophantine
+>>> from sympy import symbols
+>>> a, b, c, p, q = symbols("a, b, c, p, q", integer=True)
+>>> my_syms = (a, b, c)
+>>> pythag = a**2 + b**2 - c**2
+>>> d = diophantine(pythag, syms=my_syms)
+>>> d
+{(2*p*q, p**2 - q**2, p**2 + q**2)}
+>>> # Convert set to list
+>>> solution_list = list(d)
+>>> solution_list
+[(2*p*q, p**2 - q**2, p**2 + q**2)]
+>>> # Extract a tuple corresponding to a solution
+>>> solution_first = solution_list[0]
+>>> solution_first
+(2*p*q, p**2 - q**2, p**2 + q**2)
+>>> # Extract an expression for one variable using its order, here a is element number zero
+>>> solution_first[0]
+2*p*q
+```
+
+You can also 
```

Comment:
This needs completion

Reply:
Thanks for pointing that out. I accidentally left it over from rearranging the two sections above. So I simply deleted it just now.